modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/source.jsp

```diff
@@ -347,43 +347,38 @@ List<AssetRendererFactory<?>> classTypesAssetRendererFactories = (List<AssetRend
 					structureOptions.classList.remove('hide');
 				}
 
-				var subtypeFieldsWrappers = document.querySelectorAll('#<portlet:namespace /><%= className %>subtypeFieldsWrapper, #<portlet:namespace /><%= className %>subtypeFieldsFilterEnableWrapper');
```

Comment:
Hey @p2kmgcl ,

This change may cause a bug. ```document.getElementById('node1', 'node2')``` will return only the first node, that is why I used ```document.querySelectorAll```.

Reply:
@markocikos :man_facepalming: I didn't see the comma there, I'll send the pull again without it. Thanks!